src/Signer.php

```diff
@@ -29,33 +29,31 @@ public function getAlgorithmId(): string;
 
     /**
      * Apply changes on headers according with algorithm
```

Comment:
Removed, basically it was an over engineering. We may have signers which changes other headers (rather than `alg`) but we don't have them so far...

Thanks